src/domain/human-description/human-description.repository.interface.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  HumanDescriptions,
+  HumanDescriptionFragment,
+  HumanDescriptionTemplates,
+  ValueType,
+} from './entities/human-description.entity';
+
+export const IHumanDescriptionRepository = Symbol(
+  'IHumanDescriptionRepository',
+);
+
+export interface IHumanDescriptionRepository {
+  getDescriptions(): HumanDescriptionTemplates;
+  parseDescriptions(descriptions: HumanDescriptions): HumanDescriptionTemplates;
+
+  parseExpression(
+    valueType: ValueType,
+    valueIndex: number,
+    to: string,
+    params: unknown[],
+  ): HumanDescriptionFragment | null;
+
+  parseParam(
+    valueType: ValueType,
+    valueIndex: number,
+    to: string,
+    params: unknown[],
+  ): HumanDescriptionFragment['value'];
+}
```

Comment:
The interface is exposing functionality that is internal to how the repository works. I will create a branch with some suggestions.

Reply:
This has been resolved in [7aa7d2e](https://github.com/safe-global/safe-client-gateway-nest/pull/631/commits/7aa7d2e8d8b58ecebedd7289e7efe159a3acb191)